server/http.go

```diff
@@ -480,21 +476,34 @@ func (p *Plugin) postAuthenticationMessage(channelID string, userID string, mess
 	return p.API.SendEphemeralPost(userID, post)
 }
 
-func (p *Plugin) checkPreviousMessages(channelID string) (recentMeeting bool, meetindID int, creatorName string, err *model.AppError) {
+func (p *Plugin) checkPreviousMessages(channelID string) (recentMeeting bool, meetingLink string, creatorName string, provider string, err *model.AppError) {
 	var zoomMeetingTimeWindow int64 = 30 // 30 seconds
```

Comment:
Unrelated to this PR, but why are we checking only 30 second interval?
Maybe I'm not getting this right, but Is not it better having a state saved in the KVstore - weather or not meeting has started? We might encounter a race condition with this approach as well. No?

Reply:
I feel both solutions would suffer the same way, and I think checking the messages is clearer. Also, that way we can fetch the information directly from the post as "only source of truth", instead of taking the information from the KVStore. Finally, since the information is in the post, we can do a better inter-plugin communication. If we had things in the KVStore, it would be much more complicated this interplugin communication.